content/standards/360giving/index.html.md

```diff
@@ -1,23 +1,29 @@
 ---
-title: 360Giving Schema
-category: Domain Specific
-topic:	5.7 Grants, Loans & Subsidies
-subject: Payments Clearing and Settlement
-organisation: threesixtygiving
 name: 360Giving Schema
-status: review
-dateAdded: 2021-02-02
-dateUpdated: 2021-02-02
-classification: Domain Specific
+contraction: 360GS
+specification:
```

Comment:
When we last discussed this we agreed that 'specification' was not needed as a standalone entity right? I would suggest we flatten the structure and remove the 'name' attribute as it is redundant with the other one.